src/translations/translations_NO.json

```diff
@@ -256,6 +257,11 @@
     },
     "unpublished_resources": "Upubliserte ressurser"
   },
+  "sitemap": {
+    "contents": "Innhold",
+    "sitemap": "Nettstedskart",
+    "usage_tip": "Scroll nedover på siden, eller bruk nettleserens innebygde søk (CTRL+F) for å finne det du leter etter"
```

Comment:
ohh. hva med stakkar mac-brukere ;-)

Reply:
Skal vi få ingrid til å gå inn på UUtilsynets sitemap og se om teksten endrer seg for mac-brukere?
Hvis de ikke har noen egen forklaring for mac-brukere så tenker jeg det må være greit.

Reply:
tja. CTRL-F blir vel også feil for mobilbrukere. Kan vi ikke bare droppe ?